renku/models/_jsonld.py

```diff
@@ -399,7 +407,7 @@ def from_yaml(cls, path, client=None, commit=None):
         import yaml
 
         with path.open(mode='r') as fp:
-            source = yaml.safe_load(fp) or {}
+            source = yaml.load(fp, Loader=yaml.BaseLoader) or {}
```

Comment:
Why remove safe load? This might be a security issue.

Reply:
because you have to subclass the loader to prevent it from loading datetime strings as datetime objects - using `BaseLoader` loads everything as strings. 

from the docs: 

> BaseLoader(stream) does not resolve or support any tags and construct only basic Python objects: lists, dictionaries and Unicode strings.